lib/matplotlib/backend_bases.py

```diff
@@ -1372,6 +1372,8 @@ class MouseButton(IntEnum):
     LEFT = 1
     MIDDLE = 2
     RIGHT = 3
+    X1 = 4
```

Comment:
These should be called FORWARD and BACK and numbered 8 and 9 (see e.g. http://doc.qt.io/qt-5/qt.html#MouseButton-enum and https://wiki.archlinux.org/index.php/Mouse_buttons#Thumb_Buttons_-_forward_and_back).